.travis.yml

```diff
@@ -8,10 +8,8 @@ python:
 
 jobs:
   include:
-  - name: "Testing ansible playbook `deploy-discourse.yml`"
-    env: TEST_ROLE=discourse
-  - name: "Testing ansible playbook `deploy-matterbridge.yml`"
-    env: TEST_ROLE=matterbridge
+  - name: "Testing ansible playbook `deploy-mailserver.yml`"
+    env: TEST_ROLE=mailcow
```

Comment:
I intentionally test only this role as the other role tests are broken in master.

This is fixed in: #37

Reply:
We may need to disable this now that they are fixed